charts/kafka-operator/templates/operator-deployment-with-webhook.yaml

```
@@ -127,6 +127,9 @@ spec:
           secret:
             secretName: {{ .Values.operator.vaultSecret }}
       {{- end }}
+      {{- if .Values.additionalVolumes }}
+      {{- include "chart.additionalVolumes" . | indent 8 }}
```

This will be wrong since the indent keyword will indent the value not from the beginning of the line but from the `{` char.
```suggestion
      {{- include "chart.additionalVolumes" . | nindent 8 }}
```

---

@baluchicken : indent changed to nindent.